docs/content/preview/faq/yugabyte-managed-faq.md

```diff
@@ -4,11 +4,13 @@ linkTitle: YugabyteDB Managed FAQ
 description: YugabyteDB Managed frequently asked questions.
 headcontent:
 image: /images/section_icons/index/quick_start.png
+aliases:
+  - /preview/preview/yugabyte-cloud/cloud-faq/
 menu:
-  preview_yugabyte-cloud:
-    identifier: cloud-faq
-    parent: yugabytedb-managed
-    weight: 900
+  preview_faq:
+    identifier: yugabyte-managed-faq
```

Comment:
I suppose you could change this and the file name to "yugabytedb-managed-faq" if you're renaming anyway...